rtf_vis_tool/src/Components/AllFrustums.js

```diff
@@ -0,0 +1,115 @@
+/* Component that renders all of the Frustums using the files: cameras.txt and images.txt. Will be rendered within
+Bundle_Adj_PC.js.
+
+Author: Adi Singh
+*/
+import React, {useEffect, useState} from "react";
+import Frustum from './Frustum';
+
+// Third-Party Package Imports.
+var nj = require('numjs');
+var Quaternion = require('quaternion');
+
+// Local Imports.
+var SE3 = require('./frustum_classes/se3');
+var ViewFrustum = require('./frustum_classes/view_frustum');
+
+function AllFrustums() {
+    /* 
+    Returns:
+        A component rendering all the camera frustums.
+    */
+    const [frustumsJSX, setFrustumsJSX] = useState([]);
+
+    // Load camera intrinsics and extrinsics by fetching cameras.txt and images.txt respectively.
+    useEffect(() => {
+        
+        // Fetch camera intrinsics from cameras.txt.
+        fetch('results/ba_input/cameras.txt')
+            .then((response) => response.text())
+            .then((in_data) => {
+
+                // Fetch camera extrinsics from images.txt.
+                fetch('results/ba_input/images.txt')
+                .then((response) => response.text())
+                .then((ex_data) => {
+                    visualize_camera_frustums(in_data, ex_data)
+                });
+            });
+    }, []);
+
+    function visualize_camera_frustums(in_data, ex_data) {
+        /* Given all the intrinsic and extrinsic data for all cameras, creates an array of frustums to be 
+        rendered in react three fiber.
+
+        Args:
+            in_data (string): Raw string from cameras.txt. 
```

Comment:
would be helpful to explain that cameras.txt follows the COLMAP output convention, and link to here     Reference: https://colmap.github.io/format.html#cameras-txt